tools/create-image.sh

```diff
@@ -38,6 +95,19 @@ ssh-keygen -f $RELEASE.id_rsa -t rsa -N ''
 sudo mkdir -p $DIR/root/.ssh/
 cat $RELEASE.id_rsa.pub | sudo tee $DIR/root/.ssh/authorized_keys
 
+if [ "$TRINITY" = true ];then
```

Comment:
Do you really use trinity? I mean it's our competitor in the end :)
Why? How? What's required on syzkaller side so that you won't need trinity?

Reply:
I don't need trinity at all. The installation page mentions trinity so that I think you guys may need this in some special image creation. I will remove it in the next version.

Reply:
I see. Let's remove it then.